sequence_aligner.py

```diff
@@ -0,0 +1,13 @@
+from align import Aligner
+
+
+if __name__ == "__main__":
+    aligner = Aligner() 
+    pairs = aligner.utils.get_pairs()
+    outputs = []
+    for pair in pairs:
+        n, D = aligner.align(pair[0], pair[1])
+        alignments = aligner.utils.reconstruct_ptr(pair, D)
+        outputs.append([ alignments[0], alignments[1] , n])
```

Comment:
Minor detail, but could you make the argument a tuple instead of a list? Shouldn't matter in this case since they're treated similarly but it might be better to be sure.